src/test/scala/afenton/bazel/bsp/LspTestProcess.scala

```diff
@@ -87,8 +87,11 @@ case class BspClient(
   def buildInitialized(params: Unit): IO[DeferredSource[IO, Unit]] =
     sendRequest("build/initialized", params)
 
-  // def buildShutdown(params: Unit): IO[Unit]
-  // def buildExit(params: Unit): IO[Unit]
+  def buildShutdown(params: Unit): IO[DeferredSource[IO, Unit]] =
+    sendRequest("build/shutdown", params)
+
+  // def buildExit(params: Unit): IO[Unit] =
```

Comment:
can we remove comment

Reply:
Added TODO for it. We should be sending it, but first have to support sending `Notification`'s vs `Request`'s